packages/react/src/patterns/blocks/FeaturedLink/FeaturedLink.js

```diff
@@ -25,15 +25,18 @@ const { prefix } = settings;
  * @returns {*} FeaturedLink JSX component
  */
 const FeaturedLink = ({ heading, card }) => {
+  const cardProps = {
+    ...card,
+    cta: { ...card.cta, icon: { src: ArrowRight20, ...card.cta.icon } },
```

Comment:
@photodow Don't think this needs to be part of this review. Will make note to address for that component.

Reply:
I do feel like this is still relevant. This was changed becuase of the updates to `Card`. If it's possible to write it cleaner, simpler, and with less code I feel like that's worth exploring.

Reply:
@photodow were you thinking of something like 
```
let cardProps = card;
  cardProps.cta.icon.src = ArrowRight20;
```
?

that won't work since `icon` object needs to first be defined within the `cta` object else I get `Cannot set property 'src' of undefined`

Reply:
Is the `cardProps` variable necessary?

So I was thinking about this issue last night. No matter how you slice the dice the `cta` should be there, right? I guess you're saying `icon` might not be there? Even if we add a little validation here for `icon` I think it's still easier to read. Thoughts? Is there an edge case I might be missing here?

```javascript
if (!card.cta.icon) {
  card.cta.icon = {};
}

card.cta.icon.src = ArrowRight20;
```

Reply:
above works, but I think mutating props directly is considered bad practice which is why I used `cardProps`

Reply:
Correct me if I'm wrong, but I believe you're still mutating it because you didn't copy the object. You just assigned it to another variable.

Reply:
whoops sorry I was being confusing with
```
let cardProps = card;
  cardProps.cta.icon.src = ArrowRight20;
```  
if I use the spread operator like I kept in the PR, that will create a new object instead of a variable pointing back to the prop